ServerCore/Pages/Puzzles/FileManagement.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@page  "/{eventId}/Puzzles/FileManagement/{id}/{handler?}"
```

Comment:
Is "handler" used anywhere?

Reply:
Yes, that's matched (by convention) with OnPostDeleteAsync. That makes the handler "Delete".